### PR TITLE
created modal and textFields

### PR DESCRIPTION
**Pull request by pichar376**
first request in modal

### CODE REVIEW

File: src/pages/Delivery/components/DeliveryModal/DeliveryModal.jsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+            </>
+          );
+        }}
+      />
+    </>
+  );
+};
+
+export default DeliveryModal;
+
```
Comment (r3ivaj) [inline src/pages/Delivery/components/DeliveryModal/DeliveryModal.jsx]: maybe you could place this array into another file to not bloat this component

File: src/components/ModalBase/ModalBase.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  descriptionModal,
+  renderBody,
+  renderFooter,
+}) => {
+  const [open, setOpen] = useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+  return (
+    <div>
+      <Button onClick={handleOpen}>Open modal</Button>
```
Comment (r3ivaj) [inline src/components/ModalBase/ModalBase.jsx]: you should use the "New Delivery" button to open the modal

**Comment by r3ivaj** [comment]
the padding of the modal content doesn't match the design
Screenshot 2023-01-05 at 5 14 55 p m

File: src/pages/Delivery/components/DeliveryModal/DeliveryModal.jsx (hunk 1 above)
Comment (pichar376) [reply src/pages/Delivery/components/DeliveryModal/DeliveryModal.jsx]: Good I'll do it

File: src/components/ModalBase/ModalBase.jsx (hunk 2 above)
Comment (pichar376) [reply src/components/ModalBase/ModalBase.jsx]: Ok.

**Review by r3ivaj — APPROVED** [review]

**Pull request merged**